### ISSUE TITLE
Admin plugin

### ISSUE THREAD

**Issue by vladzima** [emilbaehr/automatic-app-landing-page#2]
Nice work!

You can integrate [jekyll-admin](https://github.com/jekyll/jekyll-admin) plugin for more convenient way of changing front matter. This would require actually serving Jekyll locally though, but for those who are comfortable with it, it would be a nice little option.

**Comment by rohanagarwal94** [comment]
Android support on a fork - https://github.com/autoswiperight/autoswiperight.github.io . Please star for karma.

**Comment by CharlieCortial** [comment]
> Android support on a fork - https://github.com/autoswiperight/autoswiperight.github.io . Please star for karma.

This is not true at all. There is nothing about Android support in the code of your repo. The requested feature isn't working on your website. Misleading and false.

**Comment by rohanagarwal94** [comment]
Hello @CharlieCortial if you opened the website - https://app.swipetinder.me/ , this is clearly for Android app. And https://github.com/autoswiperight/autoswiperight.github.io is the github repo for the code of the website.

**Comment by CharlieCortial** [comment]
Hello @rohanagarwal94 yes it is an Android app, but this Playstore feature is already part of @emilbaehr work. The requested Android Support is about getting a native Android banner for downloading the app featured on the website. It’s not the case of yours.
And yes, I checked your GitHub repo with a simple differential tool, there is no change that would allow this feature, anyway.
